app/controllers/admin/events_controller.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+module Admin
+  class EventsController < ApplicationController
+    def index
+      authorize Event
+
+      # TODO: implement this method
+      render status: 200, json: {}
+    end
```

Comment:
I created this file because we'll need it for future issues, and I just created an example action in order to test that the Pundit functionality is working properly.